### ISSUE TITLE
Problems setting up tooling

### ISSUE THREAD

**Issue by jncraig** [nvisionative/nvQuickTheme#147]
## Please summarize your question in one sentence

I'm having trouble getting all of the tooling set up, and I want to uninstall everything and start over.

## Give a more extended description

I ran into some issues and error messages getting npm, yarn, and gulp installed.  I want to remove everything and start over.  I need some help with that.  I'm also confused a bit regarding what things are installed "locally" versus what is installed "globally."

I was also getting some "lock" files created, and more than once.

So, how to I get everything cleared out?  If I delete the working nvQuickTheme directory (in skins at the moment) do I get rid of gulp and yarn, and then only need to uninstall node/npm?

In other words, rank newbie here, and I'd like to completely start over.

Also, how can I verify that things are uninstalled, and then installed correctly.

RIght now, I'm thinking that deleting directories for gulp and yarn are sufficient as these don't seem to be installed as Windows programs.  Also, clean up the Path definitions?  Then uninstall node using the windows tool?

Thanks.

**Comment by jncraig** [comment]
I'm good on a different computer.  I'll try the other one again today.

**Comment by jncraig** [comment]
I think I fixed everything on the problem computer ...

**Comment by nvisionative** [comment]
Thanks @jncraig for posting your questions here.  Setting up and configuring the various prerequisites can get messy.  We are glad to hear you were able to resolve the issues on the one computer.  If there are any tips you found useful, perhaps post those here so others can benefit from your experience.  Have a great day!
